test/Microsoft.Net.Http.Server.FunctionalTests/Utilities.cs

```diff
@@ -15,6 +17,8 @@ internal static class Utilities
         private static int NextPort = BasePort;
         private static object PortLock = new object();
 
+        internal static readonly TimeSpan DefaultTimeout = TimeSpan.FromSeconds(5);
```

Comment:
Hang detection timeouts should be high so they don't cause false positives on slow machines. at least 15 seconds.